src/index-utils.ts

```diff
@@ -632,14 +632,14 @@ export async function runViewLogics(logicResultDoc: LogicResultDoc): Promise<Log
 
   const matchingLogics = _mockable.getViewLogicsConfig().filter((viewLogicConfig) => {
     if (action === "delete") {
-      return viewLogicConfig.entity === entity && (destProp ? viewLogicConfig.destProp === destProp : true);
+      return viewLogicConfig.entity === entity && (destProp ? viewLogicConfig.destProp === `#${destProp}` : true);
     }
 
     return viewLogicConfig.actionTypes.includes(action) &&
         (
           viewLogicConfig.modifiedFields === "all" ||
             viewLogicConfig.modifiedFields.some((field) => modifiedFields.includes(field))
-        ) && viewLogicConfig.entity === entity && (destProp ? viewLogicConfig.destProp === destProp : true)
+        ) && viewLogicConfig.entity === entity && (destProp ? viewLogicConfig.destProp === `#${destProp}` : true)
```

Comment:
My initial impression when I saw this changed is that viewLogicConfig.destProp should not start with '#' sign.  It's better to investigate further why there's a '#' sign.  By tracing the problem further, you will find that the bug is actually in the init-db-structure.ts line 91 to 97.  Let me know if you agree.